tests/test_jointcal_cfht.py

```diff
@@ -105,18 +105,19 @@ def test_jointcalTask_2_visits_constrainedPoly(self):
         # NOTE: The relative RMS limit was empirically determined from the
         # first run of jointcal on this data. We should always do better than
         # this in the future!
-        dist_rms_relative = 12e-3*u.arcsecond
+        dist_rms_relative = 16e-3*u.arcsecond
+        dist_rms_absolute = 54e-3*u.arcsecond
         pa1 = None
         metrics = {'collectedAstrometryRefStars': 825,
                    'selectedAstrometryRefStars': 825,
                    'associatedAstrometryFittedStars': 2269,
                    'selectedAstrometryFittedStars': 1239,
                    'selectedAstrometryCcdImageList': 12,
-                   'astrometryFinalChi2': 1241.6,
-                   'astrometryFinalNdof': 2640,
+                   'astrometryFinalChi2': 1376.24,
+                   'astrometryFinalNdof': 2630,
```

Comment:
Do you have a comment explaining the origins of these numbers anywhere? Are they just what is expected from the test dataset you are running?

Reply:
They're all empirical: whatever the value was when I got that test to pass. They're not really something that could be pre-computed in advance (at least, not without significant difficulty). So, in that sense they function more as regression tests.

This has been a sticking point for a while: I'll add a note about them to the package readme.